config/schema.error.yaml

```
@@ -0,0 +1,575 @@
+    suggestion: 'Unable to read the `./.magento.env.yaml` file. Check file permissions.'
+    stage: post-deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::PD_CONFIG_UNABLE_TO_READ_SCHEMA_YAML:
+    title: 'Unable to read the `.schema.yaml` file'
+    suggestion: ''
+    stage: post-deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::PD_DURING_PAGE_WARM_UP:
+    step: warm-up
+    title: 'Failed to warm-up some pages'
+    suggestion: ''
+    stage: post-deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::PD_DURING_TIME_TO_FIRST_BYTE:
+    step: time-to-firs-byte
+    title: 'Failed to test time to first byte (TTFB)'
+    suggestion: ''
+    stage: post-deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::PD_CACHE_FLUSH_COMMAND_FAILED:
+    step: clean-cache
+    title: 'Command `/bin/magento cache:flush` failed'
+    suggestion: 'Check the `cloud.log` for more information. Add `VERBOSE_COMMANDS: ''-vvv''` into `.magento.env.yaml` for more detailed command output.'
+    stage: post-deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::GLOBAL_CONFIG_NOT_DEFINED:
+    title: 'Configuration isn''t defined in the `schema.yaml` file'
+    suggestion: 'Configuration isn''t defined in the `./vendor/magento/ece-tools/config/schema.yaml` file. Check that the config variable name is correct, and that it defined.'
+    stage: general
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::GLOBAL_CONFIG_PARSE_FAILED:
+    title: 'Failed to parse the `.magento.env.yaml` file'
+    suggestion: 'The `./.magento.env.yaml` file format is invalid. Use a YAML parser to check the syntax and fix any errors.'
+    stage: general
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::GLOBAL_CONFIG_UNABLE_TO_READ:
+    title: 'Unable to read the `.magento.env.yaml` file'
+    suggestion: 'Unable to read the `./.magento.env.yaml` file. Check file permissions.'
+    stage: general
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::GLOBAL_CONFIG_UNABLE_TO_READ_SCHEMA_YAML:
+    title: 'Unable to read the `.schema.yaml` file'
+    suggestion: ''
+    stage: general
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::WARN_CONFIG_PHP_NOT_EXISTS:
+    title: 'File app/etc/config.php does not exist'
+    stage: build
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_UNSUPPORTED_BUILDS_OPTION_INI:
+    title: 'The ./build_options.ini file is no longer supported'
+    stage: build
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_MISSED_MODULE_SECTION:
+    title: 'The modules section is missing from the shared config file'
+    stage: build
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_CONFIGURATION_VERSION_MISMATCH:
+    title: 'Configuration is not compatible with this version of Magento'
```

```suggestion
    title: 'The configuration is not compatible with this version of Magento'
```

Which configuration -- environment configuration, project configuration, service configuration. -- Maybe not necessary to clarify. Hard to tell without context.